spec/bindingAttributeBehaviors.js

```diff
@@ -499,7 +501,7 @@ describe('Binding attribute syntax', function() {
         expect(dataFor(testNode.childNodes[1].childNodes[0]).myCustomData).toEqual(123);
         expect(dataFor(testNode.childNodes[1].childNodes[1]).myCustomData).toEqual(123);
     });
-
+    //TODO: FAIL
```

Comment:
I have marked the tests which are failing with TODO, so that we can easily identify.